apis/events/protobuf/v1/event.proto

```diff
@@ -210,3 +216,21 @@ message ReactorEventDetails {
   // Reactor state
   string state = 2;
 }
+
+// Snapshot event details
+message SnapshotEventDetails {
+  // Parent id of the snapshot
+  string parent_id = 1;
```

Comment:
what is the parent id?

Reply:
Parent_id is replica_id. And entity_id is volume_id. Should I rename the variables?

Reply:
Yes I think would be helpful here

Reply:
Changed the variable names to replica_id and volume_id.